test/Driver/response-file.swift

```diff
@@ -24,6 +24,19 @@
 // RUN: %target-build-swift -typecheck @%t.6.resp 2>&1 | %FileCheck %s -check-prefix=RECURSIVE
 // RECURSIVE: warning: result of call to 'abs' is unused
 
+// RUN: python -c 'for i in range(500001): print "-DTEST5_" + str(i)' > %t.7.resp
+// RUN: %target-build-swift -typecheck @%t.7.resp %s 2>&1 | %FileCheck %s -check-prefix=LONG
+// LONG: warning: result of call to 'abs' is unused
+
+// RUN: python -c 'for i in range(500001): print "-DTEST6_" + str(i)' > %t.8.resp
+// RUN: %empty-directory(%t/tmp)
+// RUN: env TMPDIR=%t/tmp/ %target-swiftc_driver %s @%t.8.resp -save-temps
+// RUN: ls %t/tmp/arguments-*.resp
+
+// RUN: python -c 'for i in range(500001): print "-DTEST7_" + str(i)' > %t.9.resp
+// RUN: not %target-swiftc_driver %s @%t.9.resp -Xfrontend -debug-crash-immediately 2>&1 | %FileCheck %s -check-prefix=CRASH
+// CRASH: @{{[^ ]*}}arguments-{{[0-9a-zA-Z]+}}.resp
```

Comment:
We definitely *don't* want this behavior. The crash log should have the expanded form of the response file, not the before-expansion form. Otherwise we have no chance of reproducing the crash.

Reply:
I think it should still be possible to reproduce the crash in this situation. Temporary files are automatically saved during a crash, correct? The intermediate response files are registered as ordinary temporary files.

Reply:
That's true locally, but not very useful when a user sends us a crash log and we want to see what they were doing.